src/examples/Layout/nestedLayout.tsx

```diff
@@ -1,23 +1,29 @@
-// import { xCreateElement } from "src/lib/rezact";
 import { MasterLayout } from "./masterLayout";
 
-export function MyLayout({ router_outlet }: any) {
+export function MyLayout({ router }: any) {
+  const { $route } = router;
   return (
     <MasterLayout>
       <h1>Layout</h1>
       <ul>
-        <li>
+        <li class={$route === "/" ? "active" : ""}>
           <a href="/">Home</a>
         </li>
-        <li>
+        <li
+          class={
+            $route === "/array-state-persistent-across-route-changes"
+              ? "active"
+              : ""
+          }
+        >
           <a href="/array-state-persistent-across-route-changes">
             Home (Persistent Toggle States)
           </a>
         </li>
         <li>
           <a href="/benchmark">Benchmark</a>
         </li>
-        <li>
+        <li class={$route === "/hello-world" ? "active" : ""}>
```

Comment:
Close #64 Make Route available as a signal for applying active class to elements